tests/ci_build/test_python.sh

```diff
@@ -39,17 +43,25 @@ case "$suite" in
 
   cudf)
     source activate cudf_test
+    install_xgboost
     pytest -v -s --fulltrace -m "not mgpu" tests/python-gpu/test_from_columnar.py tests/python-gpu/test_from_cupy.py
     ;;
 
   cpu)
+    install_xgboost
     pytest -v -s --fulltrace tests/python
     cd tests/distributed
     ./runtests.sh
     ;;
 
+  cpu-py35)
+    source activate py35
+    install_xgboost
+    pytest -v -s --fulltrace tests/python
```

Comment:
Wouldn't this invoke the dask test?

Reply:
Dask tests are skipped if Dask is not installed on the system.